app/models/classification.rb

```diff
@@ -196,7 +197,7 @@ def self.get_tags_from_object(obj)
   end
 
   def self.create_category!(options)
-    self.create!(options.merge(:parent_id => 0))
+    self.create!(options.merge(:parent_id => nil))
```

Comment:
Would `self.is_category.create!(options)` work?  (completely optional, fine as-is too)

Reply:
I'm not sure if this so the caller did not have to pass in `:parent => 0` or if it was here to enforce that a `parent_id` was not passed in.

Reply:
This looks great.

1. looks like the merge is not necessary (the db default value is already taking effect)
2. Your proposed change will not override a parent_id passed in. But it looks like we don't do that anyway.

Reply:
Umm, the current `merge` does override:
```
pry(main)> {parent_id: 1}.merge({parent_id: nil})
=> {:parent_id=>nil}
pry(main)> {parent_id: 1}.merge({parent_id: 0})
=> {:parent_id=>0}
```
while the way I proposed wouldn't:
```
pry(main)> Classification.is_category.create!(name: "foo", description: "Foo")
 ...
 parent_id: 0,
 ...>
pry(main)> Classification.is_category.create!(name: "foo", description: "Foo", parent_id: 1)
 ...
 parent_id: 1,
 ...>
```
So your "not sure" question was a good question :) and maybe better not changing.

Anyway I only see 3 calls (spec/support/quota_helper.rb, plugins/manageiq-ui-classic/app/controllers/ops_controller/settings/{tags,label_tag_mapping}.rb) none of which passes parent_id.
Maybe even rewrite callers to `is_category.create!` and drop the function?
I don't care either way.